Makefile

```diff
@@ -20,22 +31,51 @@ deps-ubuntu:
 		libtesseract-dev \
 		libleptonica-dev \
 		libimage-exiftool-perl \
+		libxml2-utils \
 		tesseract-ocr-eng \
 		tesseract-ocr-deu \
 		tesseract-ocr-deu-frak
 
 # Install python deps via pip
 deps-pip:
-	pip3 install --user -r requirements.txt
+	$(PIP) install -r requirements.txt
 
 # Clone the spec dir for sample files
 spec:
 	git clone https://github.com/OCR-D/spec
```

Comment:
Just a note that the example files in spec need updating - the mets.xml should be updated to reflect recent discussions and ideally we should pick some sample images that are a) lightweight and b) for which we already have ground truth in [ocr-d.de/daten](http://ocr-d.de/daten).

Reply:
Absolutely, tracking in #41